docs/project/contributing.md

```diff
@@ -44,6 +44,7 @@ Our [Go release workflow](https://github.com/treeverse/lakeFS/blob/master/.githu
       * Apple M1 users can install this from [Azul Zulu Builds for Java JDK](https://www.azul.com/downloads/?package=jdk). Builds for Intel-based Macs are available from [java.com](https://www.java.com/en/download/help/mac_install.html).
    1. *Optional* - [PostgreSQL 11](https://www.postgresql.org/docs/11/tutorial-install.html) (useful for running and debugging locally)
    1. *Optional* - [Rust & Cargo](https://www.rust-lang.org/tools/install) (only needed if you want to build the Delta Lake diff plugin)
+   1. *Optional* - [Buf CLI](https://buf.build/docs/installation) (only needed if you like to update Protocol Buffer files)
```

Comment:
It appears there that they produce no Linux packages.  So IIUC on Linux we _still_ have to install protoc.  Maybe not for Mac/Homebrew.  Add those instructions, too?

Also, it is _always_ required to run checks-validator AFAIU.

Reply:
There are two parts the cli and the go plugin. The cli installation is curl one-liner, it is a Go project. The cli uses go run to use the plugin.
So we require less from the OS in terms of packages and depds.

About the validation - you are right, I'll need to find a better way to run this part or instruct the user in case of a change.

Reply:
Updated the code to use `go run` instead of require the use to install the tool.
It should solve the issue for running checks-validator or worried running under linux or windows.